src/nwb/ecephys/ElectricalSeries.hpp

```diff
@@ -83,15 +88,25 @@ class ElectricalSeries : public TimeSeries
    */
   std::unique_ptr<IO::BaseRecordingData> electrodesDataset;
 
-private:
-  /**
-   * @brief The neurodataType of the TimeSeries.
-   */
-  std::string neurodataType = "ElectricalSeries";
+  DEFINE_FIELD(readChannelConversion,
+               AttributeField,
+               float,
+               "data/channel_conversion",
+               Channel - specific conversion factor)
 
+  DEFINE_FIELD(readData, DatasetField, std::any, "data", Recorded voltage data)
+
+  DEFINE_FIELD(readDataUnit,
+               AttributeField,
+               std::string,
+               "data/unit",
+               Base unit of measurement for working with the data. 
+               This value is fixed to volts)
```

Comment:
`readData` and `readDataUnit` are already specified on TimeSeries. I think this should work, but just something to double-check to make sure DEFINE_FIELD successfully overwrite those inherited functions.

Reply:
The documentation displays the `readData` and `readDataUnit` descriptions as added in ElectricalSeries instead of the descriptions from the inherited TimeSeries functions. I think that suggests that DEFINE_FIELD successfully overwrote those functions, but we can definitely double check with the unit tests as well.

Reply:
Yes, double-checking with the unit tests will be useful here as the Doxygen may behave different than the actual code. To get `DEFINE_FIELD` to work with Doxygen, I had to create it is own (simplified) definition in the `Doxyfile.in` : 

https://github.com/NeurodataWithoutBorders/aqnwb/blob/6a7bb45d5f280c300763b8427a958acda6be2dfc/docs/Doxyfile.in#L10-L19

I.e., Doxygen is not using `[RegisteredType::DEFINE_FIELD](https://github.com/NeurodataWithoutBorders/aqnwb/blob/6a7bb45d5f280c300763b8427a958acda6be2dfc/src/nwb/RegisteredType.hpp#L288-L325)` when creating the documentation, so the behavior may be different. This is not ideal, but was the only way that seemed to work with Doxygen (everything else I tried to get Doxygen to actually recognize the real macro failed).

I believe the code is working correctly, but it will still be good to verify this in the unit tests.